oem/ibm/configurations/bios/enum_attrs.json

```diff
@@ -527,7 +486,8 @@
             "possible_values": ["Enabled", "Disabled", "Automatic"],
             "default_values": ["Automatic"],
             "helpText": "Controls whether or not the Runtime Processor Diagnostics (RPD) Feature will be configured on the system. Enabled ( The RPD feature will be configured regardless of the amount of installed memory), Automatic (The RPD feature will be configured on systems with at least 128GB of installed memory and not configured on systems with less than 128GB), Disabled (The RPD feature will not be configured).",
-            "displayName": "Runtime Processor Diagnostics Feature"
+            "displayName": "Runtime Processor Diagnostics Feature",
+            "readOnly": true
```

Comment:
Why readonly?

Reply:
it's a _current attribute, which can be changed only by HOST and not by any redfish client.